src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetBaseCallCoverage.java

```diff
@@ -44,7 +44,7 @@
  * </h4>
  * <h4>Input and Output</h4>
  * <p>
- * The user must indicate the target table file using the argument {@link TargetArgumentCollection#targetsFile targetsFile}.
+ * The user must indicate the target table file using the argument {@link TargetArgumentCollection#targetsFile normalizedCoveragedFile}.
```

Comment:
any chance we could be consistent with use of normalizedCoverage vs. tangentNormalizedCoverage throughout?